curator/run.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
 
+TIMES=60
+
+function waitForES() {
+  for ((i=1; i<=$TIMES; i++ )); do
+    # test for ES to be up first
+    result=$(curl --cacert $ES_CA --key $ES_CLIENT_KEY --cert $ES_CLIENT_CERT -s -w "%{http_code}" -XGET "https://$ES_HOST:$ES_PORT/" -o /dev/null)
```

Comment:
Minor tweak, but thinking of the kibana plugin issue we discussed where users MIGHT want to run un-secure.  Should we consider allowing the protocol to be added in via an undocumented ENV_VAR?